### DISCUSSION TITLE
what if im going to use this mod with already existing world?

### DISCUSSION THREAD

**Discussion by girshcatt** [Steveplays28/noisium#41] [Q&A]
What if im going to use this mod with already existing world? Will it be corrupted or will i see different terrain generation on new chunks or between new and old? (1.21.1)

**Comment by Steveplays28** [comment]
Hi @girshcatt, Noisium has 1:1 parity with world generation without Noisium. It's safe to add/remove Noisium at any time to/from existing worlds.
